blog/_posts/2015-07-17-Meet-Team-RailsGEnthusiasts.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Introducing Team RailsGEnthusiasts
+layout: blogpost
+created_at: Fri July 17 2015
+permalink: blog/_posts/2015-07-17-Meet-Team-RailsGEnthusiasts.md
+current: blog
+author: Boryana & Sophia
+twitter: RailsGEnthusia
+---
+
+
+###The Beginning of a Wonderful Journey
+
+Hello to everyone from Sofia, Bulgaria :)<br/> 
+We are Sophia and Boryana - Team RailsGirlsEnthusiasts (RailsGEnthusiasts)
+and our couches Andrew R., Marta R., Tsvety M. :)
+
+###Who we are?
+<img src="https://cloud.githubusercontent.com/assets/9285851/8754798/bf350628-2cce-11e5-9d0a-74ba118ec0d9.jpg"><br/>
+Sophia & Boryana (RailsGEnthusiasts Team)
+#####Sophia: 
+After finishing my Master degree in Economics, I have worked mostly for IT companies as I was engaged mainly with sales & marketing activities. Last year I decided that it is time to get knowledge inside, I mean to know how the software programs and systems work. That is why I took the decision to start learn programming and to learn how to create apps myself. 
+First I started to learn HTML & CSS at the beginning of 2014.
+I understood about the Rails Girls Sofia event, but I missed the one held in March 2014. I applied for the next one in October 2014. I was really very happy to see the confirmation e-mail and it was the beginning of my learning process. After the event I started to visit the local study groups in Sofia, which were organized by the main organizer of the Rails Girls Sofia event – Dimitar Dimirov. I have visited the study groups each week since then.
```

Comment:
This should probably be "I learned about" instead of "I understood about" (both translate the same in Bulgarian, but have different meaning in English :).